array_analyzer/transform/point_registration.py

```diff
@@ -188,11 +191,15 @@ def particle_filter(fiducial_coords,
         # plt.show()
 
         min_dist = np.min(dists)
-        print(min_dist)
+        mean_dist = np.mean(dists)
```

Comment:
I made this change to be consistent with the change I made to output mean particle rather than the best particle when I was troubleshooting failed spot detection. I was hoping using the mean particle (i.e. the mean of the posterior distribution) will be more robust than using the best particle (i.e. the point estimate at maximal posterior probability), but it turns out it doesn't make much difference in this case. I'll revert it back to best particle